addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -165,14 +171,130 @@ registry.category("web_tour.tours").add("pos_restaurant_sync_second_login", {
 });
 
 registry.category("web_tour.tours").add("SaveLastPreparationChangesTour", {
-        test: true,
-        url: "/pos/ui",
-        steps: () => [
-            Dialog.confirm("Open session"),
-            FloorScreen.clickTable("5"),
-            ProductScreen.clickDisplayedProduct("Coca-Cola"),
-            ProductScreen.selectedOrderlineHas("Coca-Cola", "1.0"),
-            ProductScreen.clickOrderButton(),
-            ProductScreen.orderlinesHaveNoChange()
-        ].flat(),
-    });
+    test: true,
+    url: "/pos/ui",
+    steps: () => [
+        Dialog.confirm("Open session"),
+        FloorScreen.clickTable("5"),
+        ProductScreen.clickDisplayedProduct("Coca-Cola"),
+        ProductScreen.selectedOrderlineHas("Coca-Cola", "1.0"),
+        ProductScreen.clickOrderButton(),
+        ProductScreen.orderlinesHaveNoChange()
+    ].flat(),
+});
+
+function mergeTableHelpers() {
```

Comment:
you should also add a docstring that explains that table 4 will be the parent of table 5